chapters/interfaces.txt

```diff
@@ -2752,9 +2752,16 @@ code:PatchVertices::
 Decorating a variable with the code:PatchVertices built-in decoration will
 make that variable contain the number of vertices in the input patch being
 processed by the shader.
-A single tessellation control or tessellation evaluation shader can: read
+In a Tessellation Control Shader, this is the same as the name:patchControlPoints
+member of slink:VkPipelineTessellationStateCreateInfo.
+In a Tessellation Evaluation Shader, code:PatchVertices is equivalent to the
+tessellation control output patch size, which is controlled by the code:OpExecutionMode
+code:OutputVertices (specified in either tessellation shader).
```

Comment:
```suggestion
tessellation control output patch size.
```

I don't think it really helps with understanding here to say how these things are specified, and the more places we say things like that, the more places we have to keep consistent, etc, so I'd rather leave this last clause out.

Reply:
Ok, let's leave it out.